src/app/RankStore.js

```diff
@@ -0,0 +1,12 @@
+const maxRankingSize = 3;
```

Comment:
This value is used only once, are you going to use it somewhere else?

Reply:
No, it's not used anywhere else. Should I remove it?
I thought of this as a good practice to make that kind of variables easy to change.